stubs/PyYAML/yaml/representer.pyi

```diff
@@ -1,60 +1,73 @@
+import datetime
 import sys
-from typing import Any
+from types import BuiltinFunctionType, FunctionType, ModuleType
+from typing import Any, ClassVar, NoReturn, Tuple, Type, TypeVar
 
-from yaml.error import YAMLError
+if sys.version_info >= (3, 0):
+    from collections.abc import Callable, Mapping, Sequence, Set
+else:
+    from typing import Callable, Mapping, Sequence, Set
```

Comment:
@srittau I will change it - just thought that it might be somewhat easier to clean up when the stubs for Python 2 are dropped some time in the future. But I guess this is not different from the premature optimization :smile: